### ISSUE TITLE
Im sure its a simple Troubleshoot or maybe of course something I'm missing

### ISSUE THREAD

**Issue by luisamodio** [gordonmzhu/questions#403]
### What is the associated Watch and Code lesson? 

Version 7 - Clicking “Display todos” should run todoList.displayTodos.

### Include a link to your code so I can run it on my computer. 

https://gist.github.com/luisamodio/81d3b9af566da82d24824edf20ea70ff

### Explain the exact steps I need to take to replicate the problem. 

Basically the problem is that once I click the Display ("Show All" in my file) button, the file sort of restarts and clears the memory.

### What did you expect to happen? 

I expected it to run just like yours. So to be able to have de items displayed on the command line.

### What actually happened? 

Basically the problem is that once I click the Display ("Show All" in my file) button, the file sort of restarts and clears the memory.

### Describe what you've done so far to troubleshoot the problem. 

I added an alert with the list length to the event listener to test if the button was working and it is so that doesn't seem to be the problem. Also, i tried clicking on the other buttons and text input area to see what happens and I get the same result which leads me to think that there must be some kind of configuration to prevent that from happening.

Would really appreciate your help

Thanks

**Comment by luisamodio** [comment]
Never mind. I found it!!!
The problem is that I was wrapping the button inside a form. So it refreshed...
